src/borg/archiver/create_cmd.py

```diff
@@ -185,6 +187,8 @@ def create_inner(archive, cache, fso):
                 if args.progress:
                     archive.stats.show_progress(final=True)
                 archive.stats += fso.stats
+                archive.stats.rx_bytes = getattr(repository, "rx_bytes", 0)
+                archive.stats.tx_bytes = getattr(repository, "tx_bytes", 0)
```

Comment:
shouldn't the rx/tx_bytes also use `+=`?

Reply:
No because those are the only lines where `rx_bytes` and `tx_bytes` are manipulated so they are both zero at that moment

Reply:
ok